pkg/cmd/server/origin/master.go

```diff
@@ -148,24 +148,42 @@ func (c *MasterConfig) BuildClients() {
 	c.osClient = osclient
 }
 
+// KubeClient returns the kubernetes client object
 func (c *MasterConfig) KubeClient() *kclient.Client {
 	return c.kubeClient
 }
+
+// DeploymentClient returns the deployment client object
 func (c *MasterConfig) DeploymentClient() *kclient.Client {
 	return c.kubeClient
 }
+
+// BuildLogClient returns the build log client object
 func (c *MasterConfig) BuildLogClient() *kclient.Client {
 	return c.kubeClient
 }
+
+// WebHookClient returns the webhook client object
 func (c *MasterConfig) WebHookClient() *osclient.Client {
 	return c.osClient
 }
+
+// BuildControllerClient returns the build controller client object
+func (c *MasterConfig) BuildControllerClient() *osclient.Client {
```

Comment:
because I don't want both and in the interest of your implied pending abstraction of these methods, I didn't think using the method that returned both and then assuming one of the two return values would always be the one i wanted, was the right thing to do.